src/Tribe/Service_Providers/Context.php

```diff
@@ -304,7 +334,8 @@ static function ( $post_type ) {
 					'read' => [
 						Tribe__Context::FUNC          => [
 							static function () {
-								return ! empty( tribe_get_request_var( Venue::POSTTYPE, false ) ) ?: Tribe__Context::NOT_FOUND;
+								return ! empty( tribe_get_request_var( Venue::POSTTYPE,
+									false ) ) ?: Tribe__Context::NOT_FOUND;
```

Comment:
* :no_entry_sign: Multi-line function call not indented correctly; expected 32 spaces but found 36
* :no_entry_sign: Multi-line function call not indented correctly; expected 32 spaces but found 36
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself
* :no_entry_sign: Using short ternaries is not allowed

<sub>hash: a73fbefaab4094b145dc168577660e3c</sub>